test/models/article_test.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ArticleTest < ActiveSupport::TestCase
```

Comment:
`unique_count` ve `most_recent` metotları için de test ekleyebilirsek güzel olur.